@kiva/kv-tokens/configs/kivaColors.cjs

```diff
@@ -7,6 +7,7 @@ const {
 	'dark-green': darkGreenThemeTokens,
 	mint: mintThemeTokens,
 	'dark-mint': darkMintThemeTokens,
+	brown: brownThemeTokens,
```

Comment:
Is "brown" the name being used by design?

Reply:
Nice call, I'm not sure, `Stone` maybe?

https://www.figma.com/design/61wzqlnIHQyZezhkfGkXbO/Kiva-2.0-Working-File-Q2.2204.[SH]?node-id=523-1268&m=dev

Reply:
@mcstover do you happen to know the theme name here?

Reply:
Not sure off hand but we should def get clarity from design on the name of the themes as they will eventually be showing up in Contentful so they should be "official"

Reply:
Ok, based on this slack convo https://kivahq.slack.com/archives/CB7GRAFKK/p1720567136303099?thread_ts=1720563250.654589&cid=CB7GRAFKK

Name for this theme is Dark Stone.

I agree not having an incomplete theme, but looks like there is some extra work left from design team for this.

We can go ahead with this temporary case to progress with the carousel work?